src/send/SendAmount/useTransactionCallbacks.ts

```diff
@@ -64,9 +64,9 @@ function useTransactionCallbacks({
       inputAmount: inputIsInLocalCurrency ? localAmount! : tokenAmount,
       tokenAmount,
       amountIsInLocalCurrency: inputIsInLocalCurrency,
-      tokenAddress: transferTokenAddress,
+      tokenAddress: tokenInfo!.address!,
```

Comment:
please update the types for the analytics events instead of using non-null assertions like this. the first one could even cause a runtime error, which would be unnecessary, particularly considering it's just assembling data for analytics

Reply:
bumping this